src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.search-form {
+  box-sizing: border-box;
+  position: relative;
+  display: block;
+  margin-top: 20px;
+}
+
+.search-form__input {
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+
+  padding-left: 62px;
+  font-family: Avenir-Book, sans-serif;
+  font-size: 16px;
+}
+
+.search-form__input--small {
+  height: 42px;
+  padding-left: 33px;
+  font-family: Avenir-Book, sans-serif;
+  font-size: 14px;
+}
+
+.search-form__input::placeholder {
+  color: #3d4e61;
+}
+
+.search-form__input:focus::placeholder {
+  text-shadow: none;
+  font-family: Avenir-Book, sans-serif;
```

Comment:
Include a fallback generic font family, such as 'sans-serif', in case the custom font doesn't load.